### PR TITLE
Add usernames for Persian translation reviewers

### PR DESCRIPTION
**Pull request by hamidrezakp**
Hello,
This list is still a Draft, will be ready in a day or two.

### CODE REVIEW

**Comment by mgeisler** [comment]
This should eventually be reviewed by @henrif75 who is managing the translation community.

**Review by henrif75 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,7 +4,7 @@ po/da.po @mgeisler @voss @thedataking
 po/de.po @ronaldfw @fechu
 po/el.po @Mandragorian
 po/es.po @deavid @JaviSorribes
-po/fa.po @DannyRavi
+po/fa.po @DannyRavi @hamidrezakp @moaminsharifi @mehrdad77
```
Comment (henrif75) [inline .github/CODEOWNERS:7]: Can you check the user @mehrdad77? Apparently it's not an existing username.

**Review by henrif75 — CHANGES_REQUESTED** [review]
Check user @mehrdad77

File: .github/CODEOWNERS (hunk 1 above)
Comment (hamidrezakp) [reply .github/CODEOWNERS:7]: Ops that's my fault, it was `mehrad77`

**Review by henrif75 — APPROVED** [review]
Thanks!

**Pull request merged**